api_tests/src/actors/swap_factory.ts

```diff
@@ -19,18 +19,36 @@ import {
     Herc20EthereumErc20RequestParams,
     Peer,
 } from "comit-sdk";
+import { HarnessGlobal } from "../utils";
+
+declare var global: HarnessGlobal;
 
 export default class SwapFactory {
     public static async newSwap(
         alice: Actor,
-        bob: Actor
-    ): Promise<
-        [
-            HanEthereumEtherHalightLightningBitcoinRequestBody,
-            HanEthereumEtherHalightLightningBitcoinRequestBody
-        ]
-    > {
-        const ledgers: (keyof AllWallets)[] = ["ethereum", "lightning"];
+        bob: Actor,
+        dry?: boolean
```

Comment:
I am not seeing any use of the `dry` parameter?

Reply:
in `lightning_routes.ts`